src/Kunstmaan/AdminBundle/Tests/unit/EventListener/ConsoleExceptionListenerTest.php

```diff
@@ -7,29 +7,32 @@
 use Kunstmaan\AdminBundle\EventListener\ConsoleExceptionListener;
 use PHPUnit_Framework_TestCase;
 use Psr\Log\LoggerInterface;
-use Symfony\Component\Console\Event\ConsoleErrorEvent;
 use Symfony\Component\Console\Event\ConsoleExceptionEvent;
+use Symfony\Component\HttpKernel\Kernel;
 
 class ConsoleExceptionListenerTest extends PHPUnit_Framework_TestCase
 {
+    /**
+     * @group legacy
+     */
     public function testListener()
     {
+        if (Kernel::VERSION_ID >= 4000) {
```

Comment:
```suggestion
        if (Kernel::VERSION_ID >= 40000) {
```

4000 -> 40000 otherwise test is also skipped on sf3.4